src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -226,6 +226,41 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class CallbackContext:
```

Comment:
some documentations on purpose and usage would be nice both for this one and the CommissioningContext below.